rbx_dom_lua/src/PropertyDescriptor.lua

```diff
@@ -20,7 +20,10 @@ local function set(container, key, value)
 end
 
 function PropertyDescriptor.fromRaw(data, className, propertyName)
+	local dataType, ty = next(data.DataType)
```

Comment:
I think we should rename these variables and leave a comment as this is unintuitive.

The first value is only _actually_ `dataType` if it's equal to `Enum`, so we should just call it `key`, and the second value `value`. We should also mention in a comment about the two cases we're expecting:
- `{ Enum = "<name of enum>" }`
- `{ Value = "<data type>" }`